core/arch/x86/decode_table.c

```diff
@@ -1535,13 +1551,18 @@ const instr_info_t * const op_instr[] =
 #define KVd TYPE_K_VEX, OPSZ_4
 #define KVq TYPE_K_VEX, OPSZ_8
 #define KE1b TYPE_K_EVEX, OPSZ_1b
+#define KE2b TYPE_K_EVEX, OPSZ_2b
+#define KE4b TYPE_K_EVEX, OPSZ_4b
 #define KEb TYPE_K_EVEX, OPSZ_1
 #define KEw TYPE_K_EVEX, OPSZ_2
 #define KEd TYPE_K_EVEX, OPSZ_4
 #define KEq TYPE_K_EVEX, OPSZ_8
 #define Ves TYPE_V, OPSZ_16_vex32_evex64
 #define Ved TYPE_V, OPSZ_16_vex32_evex64
 #define Vqq_oq TYPE_V, OPSZ_vex32_evex64
```

Comment:
So this should be a single code with no underscore -- `Vf` or sthg.

Reply:
Yep, see comment above and in commit message.

Reply:
(2 existing ones of the H and W class were fixed)